packages/ui/src/components/Markdown/customDirectives.tsx

```diff
@@ -0,0 +1,22 @@
+import { visit, Node, VisitorResult } from 'unist-util-visit'
+
+
+export default function customDirectives () {
+    return transform
+
+    function transform (tree: Node) {
+        return visit(tree, ['textDirective', 'leafDirective', 'containerDirective'], ondirective)
+    }
+
+    function ondirective (node: any): VisitorResult {
+        // console.log('node', node)
+        const { name, attributes, children } = node
+        const text = children[0]?.value ?? ''
+
+        const data = node.data || (node.data = {})
+        data.hName = 'section'
+        data.hProperties = { ...attributes, text, type: name }
+        console.log('data', data)
```

Comment:
Please remove debug outputs like this.
or use
```suggestion
        console.debug('data', data)
```
if it is really necessary.